src/docs/metrics/social-volume/index.md

```diff
@@ -1,197 +1,164 @@
-
-By default we use an auto-generated search query to track the volume of mentions for each project, e.g.: `btc OR bitcoin NOT cash NOT gold NOT abc ...etc`. These queries can be manually modified by our admins to exclude redundant results or include some additional word to the query if necessary.
-
-### How to Access
-
-#### [Sanbase](https://app.santiment.net)
-
-The metric is available on Sanbase:
-
-- on our charts **for any selected asset**:
-
-  <iframe frameborder="0" height="340" src="https://app.santiment.net/chart?from=2019-06-02T21%3A00%3A00.000Z&interval=12h&isShowAnomalies=true&metrics=historyPrice,socialVolume&scale=auto&slug=santiment&title=Santiment%20%28SAN%29&to=2019-12-03T21%3A00%3A00.000Z&viewOnly=true"></iframe>
-
-- in the [Labs](https://app.santiment.net/labs/trends) section for **any desired search query**.
-
-#### [Sanbase Graphs](https://graphs.santiment.net/social)
-
-The metric is available **for any selected asset**.
-
-#### [SanAPI](https://neuro.santiment.net/)
-
-The metric is available on SanAPI.
-
-#### [Sansheets](https://sheets.santiment.net/)
-
-The metric is available on Sansheets.
-
-### Availability
-
-|                | Free               | Basic              | Pro                | Premium            | Enterprise         |
-| -------------- | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
-| Sanbase        | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| Sanbase Graphs | :x:                | :x:                | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| SanAPI         | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| Sansheets      | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+The total number of [text documents](/metrics/details/text-document) that
+contain the given search term at least once. Examples of documents are telegram
+messages and reddit posts. If a single short telegram message includes the word
+`crypto` more than once, this message will increase the social volume of the
+word `crypto` by 1. If a long reddit post contains the word `crypto` 10 times,
+this again will increase the social volume of the word `crypto` by 1.
+
+Social Volume can be computed for an [asset](/glossary#asset) or for arbitrary
+search terms. When an asset is used, a special search term is constructed. For
+example querying the social volume for `santiment` will result in a search term
+similar to:
+
+> (SAN OR santiment OR santoken OR santimentnet OR sancoin OR sansan OR sancoin)
+> AND NOT ((san AND francisco) OR (san AND fran) OR (san AND diego) OR (san AND
+> marino) OR (san AND jose))
+
+which includes documents containg `santiment` or `san` but excludes documents
```

Comment:
I would put here more explanations, Document can can contain not only 'santiment',  but any (at least one) search term from `st line, but not a part of 'not' section.

Reply:
:+1: